UOP1_Project/ProjectSettings/ProjectVersion.txt

```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 2019.4.7f1
-m_EditorVersionWithRevision: 2019.4.7f1 (e992b1a16e65)
+m_EditorVersion: 2020.1.6f1
+m_EditorVersionWithRevision: 2020.1.6f1 (fc477ca6df10)
```

Comment:
Please don't update the editor version, this project is meant to be made in 2019.4 LTS